src/main/java/io/vertx/core/net/ProxyType.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * 
+ */
+package io.vertx.core.net;
+
+/**
+ * @author <a href="http://oss.lehmann.cx/">Alexander Lehmann</a>
+ *
+ */
+public enum ProxyType {
```

Comment:
Missing @VertxGen and javadoc for each type, so it will generate a nice data sheet.